apps/account-worker/src/transaction_publisher/publisher.service.ts

```diff
@@ -196,6 +203,26 @@ export class TransactionPublisherService extends BaseConsumer implements OnAppli
     }
   }
 
+  async processProxyTxn(
+    ext: SubmittableExtrinsic<'promise', ISubmittableResult>,
+    accountId: string,
+    signer: Signer,
+  ): Promise<[SubmittableExtrinsic<'promise'>, HexString]> {
+    try {
+      const { nonce } = await this.blockchainService.api.query.system.account(accountId);
```

Comment:
Should this use `nonceService`? We might need to add a new method in there that gets accountId as a param

Reply:
Decided that this was out of scope.